pkg/io/io_test.go

```diff
@@ -0,0 +1,111 @@
+package io
+
+import (
+	"testing"
+
+	"github.com/fusor/cpma/pkg/env"
+	"github.com/fusor/cpma/pkg/io/remotehost"
+	legacyconfigv1 "github.com/openshift/api/legacyconfig/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Save before overriding
+var _RunCMD = remotehost.RunCMD
+
+// Overriding
+func mockRunCMD(hostname, cmd string) (string, error) {
+	return "remote value", nil
+}
+
+func TestFetchFile(t *testing.T) {
+	testCases := []struct {
+		name     string
+		remote   bool
+		expected []byte
+		filename string
+	}{
+		{
+			name:     "Fetch from remote",
+			remote:   true,
+			filename: "remote-file",
+			expected: []byte{0x72, 0x65, 0x6d, 0x6f, 0x74, 0x65, 0x20, 0x76, 0x61, 0x6c, 0x75, 0x65},
```

Comment:
Not sure that everyone will be able to comprehend the HEX codes there. Wouldn't `string` value be better?

Reply:
Fair enough!